javascripts/govuk/multivariate-test.js

```diff
@@ -77,7 +80,17 @@
       cohort, 
       2 // session level
     ); 
-    // Fire off a dummy event to set the custom var on the page.
+  };
+
+  MultivariateTest.prototype.setUpContentExperiment = function(cohort) {
+    if(this['contentExperimentId'] && this['cohorts'][cohort]['variantId']){
+      window.ga('set', 'expId', this['contentExperimentId']);   
+      window.ga('set', 'expVar', this['cohorts'][cohort]['variantId']);
```

Comment:
We should check somewhere that `window.ga` exists. eg Users with adblock can block GA from loading, or the GA library could fail to load. The universal tracker handles this by proxying the`ga` call: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/javascripts/govuk/analytics/google-analytics-universal-tracker.js#L97-L101